### PR TITLE
fix: correct width issue caused by tooltip addition in SingleSelect

### LINKED ISSUE #4031: Bug: width problem in AppSideBar CSS causing layout issues
### Description:

There is an issue with the `AppSideBar `component where the width is incorrectly set in the CSS, leading to layout problems. 

Screenshot_25-8-2024_131842_app opensauced pizza

### Suggested Fix:

I have corrected this issue in a pull request. The CSS width properties in  `AppSideBar.js` have been adjusted to match the intended design specifications.

### PR DESCRIPTION
**Pull request by mouad-dadda**
## Description

a layout issue in the `AppSideBar` component caused by a recent change to the `SingleSelect` component. Specifically, when a tooltip for long titles was added to `SingleSelect`, a width was set that inadvertently disrupted the layout.

## Related Tickets & Documents

Fixes #4031 

## Mobile & Desktop Screenshots/Recordings

Problem :

Screenshot_25-8-2024_131842_app opensauced pizza

layout issiu

fixed 💯 : 
fixedall

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

### CODE REVIEW

**Comment by nickytonline** [comment]
One other thing to notw @mouad-dadda is that the plus icon is too dark. It should be the same colour as the top level sidebar menu items.

**Comment by mouad-dadda** [comment]
Got it, @nickytonline :

image

**Comment by jpmcb** [comment]
Approved netlify run, approved GitHub actions, requested engineering review.

File: components/atoms/Select/single-select.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const SingleSelect = ({
         )}
       >
         <Tooltip content={current?.label ?? placeholder}>
-          <div className="flex items-center w-fit max-w-48">
+          <div className="flex items-center  w-44 ">
```
Comment (jpmcb) [inline components/atoms/Select/single-select.tsx:67]: nit: Extra space here

```suggestion
          <div className="flex items-center w-44 ">
```

**Review by jpmcb — APPROVED** [review]
Screenshot 2024-08-28 at 12 19 51 PM

Looks solid: small nit comment that should be addressed

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**